translate/compiler_internal_test.go

```diff
@@ -29,6 +46,125 @@ func Test_compileStatement(t *testing.T) {
 }
 
 // TODO: implement test cases :-)
-func Test_generateFuncJumper(t *testing.T) {
+func Test_compileAssignStatement(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileReturnStatement(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileIfStatement(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileBlockStatement(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileExpressionStatement(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileFunctionLiteral(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileExpression(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileCallExpression(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileInfixExpression(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compilePrefixExpression(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileIntegerLiteral(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileStringLiteral(t *testing.T) {
+
+}
+
+// TODO: implement test cases :-)
+func Test_compileBooleanLiteral(t *testing.T) {
+	tests := []struct {
+		expression *ast.BooleanLiteral
+		expected   Bytecode
+	}{
+		{
+			expression: &ast.BooleanLiteral{
+				Value: false,
+			},
+			expected: Bytecode{
+				RawByte: []byte{0x21, 0x00, 0x00, 0x00, 0x00},
+				AsmCode: []string{"Push", "00000000"},
+				Abi:     abi.ABI{},
+			},
+		},
+		{
+			expression: &ast.BooleanLiteral{
+				Value: true,
+			},
+			expected: Bytecode{
+				RawByte: []byte{0x21, 0x00, 0x00, 0x00, 0x01},
+				AsmCode: []string{"Push", "00000001"},
+				Abi:     abi.ABI{},
+			},
+		},
+	}
+
+	for i, test := range tests {
+		bytecode := &Bytecode{
+			RawByte: make([]byte, 0),
+			AsmCode: make([]string, 0),
+			Abi:     abi.ABI{},
+		}
+		err := compileBooleanLiteral(test.expression, bytecode)
+
+		if err != nil {
+			t.Fatalf("test[%d] - compileBooleanLiteral() had error. err=%v", i, err)
```

Comment:
We need to test whether function returns valid error or not.

Reply:
But, the error is came out from just `encoding.EncodeOperand()`  not `compileBooleanLiteral()`. So, I think that's enough... Do you think it need?